src/content/developers/docs/intro-to-ether/index.md

```diff
@@ -74,7 +70,7 @@ Users can query the ether balance of any [account](/developers/docs/accounts/) b
 
 ## Further reading {#further-reading}
 
+- [Defining Ether and Ethereum](https://www.cmegroup.com/education/courses/introduction-to-ether/defining-ether-and-ethereum.html) – _CME Group_
 - [Ethereum Whitepaper](/whitepaper/): The original proposal for Ethereum. This document includes a description of ether and the motivations behind its creation.
-- [Ethereum Yellowpaper](https://ethereum.github.io/yellowpaper/paper.pdf): A technical description of Ethereum's implementation. This document describes the underlying logic of the Ethereum blockchain and ether cryptocurrency.
```

Comment:
Any reason in particular you want to remove this?

Reply:
Also curious on this

Reply:
@corwintines @wackerow 

Seems a bit crazy to me to expect a user learning about ether to be able to understand the yellow paper at this point in their journey? It's a bit like - ok you've learned what ether is now go build an Ethereum client.

Personally I'd like to push this back to at least the end of the foundational topics.

Reply:
I think that makes sense. Where were you thinking about putting it instead in the foundational topics?

Reply:
Paul, Sam and me discussed yday - we think EVM is a good fit

Reply:
updateIntroToEther